.github/workflows/deploy.yml

```diff
@@ -99,3 +99,22 @@ jobs:
             -d "$(jq -n --arg version "$packageVersion" '{current: $version}')" \
             -y --project $gcloudProject \
             "/versionInfo/${packageName}"
+
+      # Update Firestore "email_templates" collection with contents of email templates
+      - name: Update Email Templates
```

Comment:
This stage updates email templates in Firestore after app is successfully deployed - it is dynamic to environment, so this will keep them up to date in all environments